bindings/python/docstrings.inl

```diff
@@ -0,0 +1,462 @@
+#pragma once
+#include <string>
+namespace manifold_docs {
+const auto cross_section__area_f5458809be32 = R"___(Return the total area covered by complex polygons making up the
```

Comment:
Is this file generated by your script? Would be great to have some documentation about how the system works and the right way to maintain it. Looks promising though!

Reply:
Generated a comment in the file, and added comments in the gen_docs.py script.